src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,84 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number
+  perPage: number
+  currentPage: number
+  onPageChange: (page: number) => void
+};
+
+export const Pagination: React.FC<Props> = (
+  {
+    total,
+    perPage,
+    currentPage,
+    onPageChange,
+  },
+) => {
+  const items = getNumbers(1, Math.ceil(total / perPage))
+    .map(n => n);
+
+  const checkPrevLink = currentPage === 1;
+  const checkNextLink = items.length === currentPage;
+
+  return (
+    <ul className="pagination">
+      <li className={classNames(
+        'page-item',
+        { disabled: checkPrevLink },
+      )}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={checkPrevLink}
```

Comment:
it should be functionally disabled as well (not only look like disabled) - reason of tests fail (check task requirements)